lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -91,6 +91,11 @@ class OutlineTabView(RetrieveAPIView):
             analytics_id: (str) The unique id given to the tool.
             title: (str) The display title of the tool.
             url: (str) The link to access the tool.
+        dates_banner_info: (obj)
+            missed_deadlines: (bool) Whether the user has missed any graded content deadlines for the given course.
+            missed_gated_content: (bool) Whether the user has missed any gated content for the given course.
+            content_type_gating_enabled: (bool) Whether content type gating is enabled for this enrollment.
+            verified_upgrade_link: (str) The URL to ecommerce IDA for purchasing the verified upgrade.
```

Comment:
So you added in this documentation, but didn't add it to the serializer or data. What's going on?

Reply:
Oh RIGHT. The whole implicit mixin thing. Woof. Okay. In that case, I just want to compliment you on your documentation!